### PR TITLE
chore(SplitChunksPlugin): update defaults

### PR DESCRIPTION
**Pull request by byzyk**
As per [card](https://github.com/webpack/webpack/projects/5#card-10750335), bumping defaults for `SplitChunksPlugin`

**What kind of change does this PR introduce?**

feature

**Did you add tests for your changes?**

N/A

**Does this PR introduce a breaking change?**

Yes

**What needs to be documented once your changes are merged?**

Default values are not mentioned in docs.

### CODE REVIEW

File: lib/optimize/SplitChunksPlugin.js [hunk 1]
```diff
@@ -105,8 +105,8 @@ module.exports = class SplitChunksPlugin {
 			),
 			minSize: options.minSize || 0,
 			minChunks: options.minChunks || 1,
-			maxAsyncRequests: options.maxAsyncRequests || 1,
-			maxInitialRequests: options.maxInitialRequests || 1,
+			maxAsyncRequests: options.maxAsyncRequests || 6,
+			maxInitialRequests: options.maxInitialRequests || 4,
```
Comment (byzyk) [inline lib/optimize/SplitChunksPlugin.js]: Not sure if the above changes are needed since we already do that in `WebpackOptionsDefaulter`.

/cc @ooflorent

Comment (ooflorent) [reply lib/optimize/SplitChunksPlugin.js]: No. Leave them untouched.

**Comment by sokra** [comment]
Thanks

**Comment by edmorley** [comment]
> Default values are not mentioned in docs.

They are mentioned here:
https://webpack.js.org/plugins/split-chunks-plugin/#defaults
https://webpack.js.org/plugins/split-chunks-plugin/#optimization-splitchunks

**Comment by byzyk** [comment]
@edmorley good catch, thanks! We'll have to update this in v5 docs.

**Comment by montogeek** [comment]
@byzyk Please open an issue on webpack docs repo

**Pull request merged**